File: 02. Algorithms/01. Arrays/25. Array Partition 1/array-partition-i.cpp
class Solution {
public:
    int arrayPairSum(vector<int>& nums) {
        int n = nums.size();
         sort(nums.begin(),nums.end());
        int total=0;
        for(int i=0;i<n;i++)
        {
            if(i%2==0)
            {
                total += min(nums[i],nums[i+1]);
            }
        }
        return total;
    }
};

File: 02. Algorithms/01. Arrays/25. Array Partition 1/array-partition-i.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var arrayPairSum = function(nums) {
    nums = nums.sort((a,b) => a-b);
    let max = 0;

    for(let i = 0; i < nums.length; i += 2) {
        max += Math.min(nums[i], nums[i+1]);
    }
    return max;
};
